### ISSUE TITLE
Create documentation using lookatme (or markata?)

### ISSUE THREAD

**Issue by d0c-s4vage** [d0c-s4vage/lookatme#155]
**Describe the Feature Request**
Documentation should be created using lookatme. This could be a prototype for, or maybe directly depend on #154 - Enable html export of lookatme slides.

Using lookatme to generate lookatme's documentation would be so very fitting.

Side note: markata has a very similar format to lookatme, and already documents its own code with itself. Unsure how far I want to take this, but at least the prose/long-form documentation?

**Comment by d0c-s4vage** [comment]
Closing this in favor of #196
